search-app/src/cmr/search/site/routes.clj

```diff
@@ -2,19 +2,31 @@
   "This namespace is the one responsible for the routes intended for human
   consumption."
   (:require
-   ;; Third-party libs
-   [compojure.core :refer :all]
-   [ring.swagger.ui :as ring-swagger-ui]
+    ;; Third-party libs
+    [compojure.core :refer :all]
```

Comment:
Oh, I think the latest code also removes the old sorting of req's and just puts them altogether in alpha order.